demos/tests/cases.py

```diff
@@ -600,12 +600,31 @@ def single_option_cases(key, *args):
         [
             *combine_cases(
                 TestCase(options={'--architecture_type': 'centernet'}),
-                single_option_cases('-m',
-                    ModelArg('ctdet_coco_dlav0_384'),
-                    ModelArg('ctdet_coco_dlav0_512')),
+                [
+                    *single_option_cases('-m',
+                        ModelArg('ctdet_coco_dlav0_384'),
+                        ModelArg('ctdet_coco_dlav0_512')
+                    ),
+                    *combine_cases(
+                        TestCase(options={
+                            '--mean_values': ['104.04', '113.985', '119.85'],
+                            '--scale_values': ['73.695', '69.87', '70.89']
+                        }),
+                        single_option_cases('-m',
+                            ModelOnnxArg('ctdet_coco_dlav0_384'),
+                            ModelOnnxArg('ctdet_coco_dlav0_512')
+                        ),
+                    ),
+                ]
             ),
-            TestCase(options={'--architecture_type': 'faceboxes',
-                              '-m': ModelArg('faceboxes-pytorch')}
+            *combine_cases(
+                TestCase(options={'--architecture_type': 'faceboxes'}),
+                [
+                    *single_option_cases('-m',
+                        ModelArg('faceboxes-pytorch')
+                    ),
```

Comment:
`single_option_cases` with one argument is redundant; you can just replace it with a `TestCase`.

Reply:
Updated